src/public/js/video-tagging/video-tagging.html

```diff
@@ -986,6 +1023,41 @@
         this.rotation = (this.rotation + 90) % 360;
         $('#vid').css("transform",`rotate(${this.rotation}deg)`);   
     },
+    seekToFrame: function(id, fireEvents = true) {
+        if (this.checkRegionLabels()){
+            //raise before next frame  
+            if (this.imagelist){//image handling
+                if(this.imageIndex == id - 1) {
+                    //Only fire before step if seeking to the next frame
+                    if(fireEvents)$('#video-tagging').trigger('stepFwdClicked-BeforeStep');    
+                }
+                if(this.imageIndex == id + 1) {
+                    //Only fire before step if seeking to the previous frame
+                    if(fireEvents)$('#video-tagging').trigger('stepBwdClicked-BeforeStep');    
+                }
+                if (id < this.imagelist.length-1) {
```

Comment:
What happens if id is negative? Do we need to check if id >=0?

Reply:
id must be >= 0, it's tied to the imagelist index which is zero bound